right/src/ledmap.c

```diff
@@ -505,6 +505,10 @@ static rgb_t* determineFunctionalRgb(key_action_t* keyAction) {
         case KeyActionType_PlayMacro:
             keyActionColor = KeyActionColor_Macro;
             break;
+        case KeyActionType_Connections:
+        case KeyActionType_Other:
+            keyActionColor = KeyActionColor_Special;
```

Comment:
@kareltucek  Would we like to keep it as Special or we rename it to Device too? It is up to you I just follow the same namings in Agent

Reply:
Name it device.